speech/tests/unit/test__gax.py

```diff
@@ -16,6 +16,9 @@
 
 import mock
 
+from google.cloud import _helpers
+from google.cloud.gapic.speech.v1.speech_client import SpeechClient
```

Comment:
FWIW it's not my shtick, I just want to keep the style intact. If we decide to break from the style, it should be wholesale, not ad-hoc.